### PR TITLE
Fix checkbox

### PR DESCRIPTION
**Pull request by rarila**
:checkered_flag: With a narrow margin in front of the :koala: …
… here's the winner of the Derp-A-Derp-Award :full_moon: :ribbon:

### CODE REVIEW

File: app/view/twig/_macro/_buic.twig [hunk 1]
```diff
@@ -27,7 +26,7 @@
             'value':             '1',
         } %}
 
-        <div{{ hclass(['buic-checkbox', opt.class|default()]) }}>
```
Comment (CarsonF) [inline app/view/twig/_macro/_buic.twig]: Are there _bolt-widgets_ that are not _buic_?

File: app/view/twig/_macro/_buic.twig [hunk 2]
```diff
@@ -27,7 +26,7 @@
...
+        <div{{ hclass(['buic-checkbox', opt.class|default()]) }} data-bolt-widget="buicCheckbox">
```
Comment (CarsonF) [inline app/view/twig/_macro/_buic.twig]: Are there bolt-widgets that are not buic?

Comment (rarila) [inline app/view/twig/_macro/_buic.twig]: yep

Comment (rarila) [inline app/view/twig/_macro/_buic.twig]: See [folders](https://github.com/bolt/bolt/tree/master/app/src/js/widgets)

**Pull request merged**